nodejs_container/index.js

```diff
@@ -4,52 +4,25 @@ const fs = require('fs');
 const path = require('path');
 
 // deals with ensuring the correct version for the machine/node version
-const binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
-
-const HolochainApp = require(binding_path).HolochainApp;
+const binding_path = binary.find(path.resolve(path.join(__dirname, './package.json')));
+
+const { ConfigBuilder, Habitat } = require(binding_path);
+
+Habitat.prototype.callRaw = Habitat.prototype.call
+Habitat.prototype.call = function (id, zome, trait, fn, params) {
+    const stringInput = JSON.stringify(params);
+    const rawResult = this.callRaw(id, zome, trait, fn, stringInput);
+    let result;
+    try {
+        result = JSON.parse(rawResult);
+    } catch (e) {
+        console.log("JSON.parse failed to parse the result. The raw value is: ", rawResult);
+        result = { error: "JSON.parse failed to parse the result", rawResult };
+    }
+    return result;
+}
```

Comment:
This is a replacement for the old `_call` / `call` switcheroo, just renamed `_call` to `callRaw`